packages/website/docs/creating-a-renderer.md

```diff
@@ -331,8 +332,8 @@ function Autocomplete() {
 }
 ```
 
-Users will now feel a little bit closer to what they're used to on mobile apps.
+This way, users feel a bit closer to what they're used to on mobile apps.
 
----
+## Help and discussion
 
-You now have enough knowledge to build your own experience based on Autocomplete. If you feel like some topics weren't covered in this page, feel free to [open an issue](https://github.com/algolia/autocomplete.js/issues/new).
+You now have enough knowledge to build your own experience based on Autocomplete. If you find that some topics weren't covered, feel free to [open an issue](https://github.com/algolia/autocomplete.js/issues/new).
```

Comment:
This conclusion might not be totally aligned with the other guides now?

Reply:
True. I was thinking we may want to call this out especially in this tutorial. The content of this tutorial leads more easily to feature requests than the other ones, so I'm ok with having a slightly different ending. (This instead of "next steps".) WDYT

Reply:
Good to me!